MetaData/python/JobConfig.py

```diff
@@ -166,6 +166,11 @@ def customize(self,process):
                             wei *= xsec.get("br",1.)
                             wei *= xsec.get("kf",1.)
                             obj.lumiWeight = wei
+                    if hasattr(obj,"intLumi"):
+                        if isdata:
+                            obj.intLumi=self.targetLumi #FIXME is this where this info should come from for data?
```

Comment:
For data the `IntLumi` parameter is not used, so we can even set it to 0 here, so that the downstream scripts get into error if they try to use it for normalization.